From foo@baz Fri Sep 11 09:51:31 PDT 2009
Date: Fri, 11 Sep 2009 09:51:31 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: vme: change to VME_BUS

Turns out the m68k arch already has a CONFIG_VME, so use
CONFIG_VME_BUS instead.

Thanks to Geet Uytterhoeven for pointing this out.

Cc: Geert Uytterhoeven <geert@linux-m68k.org>
Cc: Martyn Welch <martyn.welch@gefanuc.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/Makefile     |    2 +-
 drivers/staging/vme/Kconfig  |    4 ++--
 drivers/staging/vme/Makefile |    2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -50,4 +50,4 @@ obj-$(CONFIG_USB_CPC)		+= cpc-usb/
 obj-$(CONFIG_RDC_17F3101X)	+= pata_rdc/
 obj-$(CONFIG_FB_UDL)		+= udlfb/
 obj-$(CONFIG_HYPERV)		+= hv/
-obj-$(CONFIG_VME)		+= vme/
+obj-$(CONFIG_VME_BUS)		+= vme/
--- a/drivers/staging/vme/Kconfig
+++ b/drivers/staging/vme/Kconfig
@@ -2,13 +2,13 @@
 # VME configuration.
 #
 
-menuconfig VME
+menuconfig VME_BUS
 	tristate "VME bridge support"
 	depends on PCI
 	---help---
 	  If you say Y here you get support for the VME bridge Framework.
 
-if VME
+if VME_BUS
 
 source "drivers/staging/vme/bridges/Kconfig"
 
--- a/drivers/staging/vme/Makefile
+++ b/drivers/staging/vme/Makefile
@@ -1,7 +1,7 @@
 #
 # Makefile for the VME bridge device drivers.
 #
-obj-$(CONFIG_VME)		+= vme.o
+obj-$(CONFIG_VME_BUS)		+= vme.o
 
 obj-y				+= bridges/
 obj-y				+= devices/
